sktime/forecasting/online_ensemble.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+from sktime.forecasting.compose._ensemble import EnsembleForecaster
+from sktime.forecasting.model_selection import OnlineSplitter
+from .ensemble_algorithms import EnsembleAlgorithms
+
+
+class OnlineEnsembleForecaster(EnsembleForecaster):
+    """Online Updating Ensemble of forecasters
+
+    Parameters
+    ----------
+    ensemble_algorithm : ensemble algorithm
+    forecasters : list of (str, estimator) tuples
+    n_jobs : int or None, optional (default=None)
+        The number of jobs to run in parallel for fit. None means 1 unless
+        in a joblib.parallel_backend context.
+        -1 means using all processors.
+    """
+
+    _required_parameters = ["forecasters"]
+
+    def __init__(self, forecasters, ensemble_algorithm=None, n_jobs=None):
+        self.n_jobs = n_jobs
+        if ensemble_algorithm is None:
+            self.ensemble_algorithm = EnsembleAlgorithms(len(forecasters))
+        else:
+            self.ensemble_algorithm = ensemble_algorithm
+
+#         if self.ensemble_algorithm.n != len(forecasters):
+#             raise ValueError("Number of Experts in Ensemble Algorithm \
+#                              doesn't equal number of Forecasters")
+
+        super(EnsembleForecaster, self).__init__(forecasters=forecasters,
+                                                 n_jobs=n_jobs)
+
+    def _fit_ensemble(self, y_val, X_val=None):
+        """Fits the ensemble by allowing forecasters to predict and
+           compares to the actual parameters.
+
+        Parameters
+        ----------
+        y_val : pd.Series
+            Target time series to which to fit the forecaster.
+        X_val : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored
+        """
+        fh = np.arange(len(y_val)) + 1
+        expert_predictions = np.column_stack(self._predict_forecasters(
+                                             fh=fh, X=X_val))
+        actual_values = np.array(y_val)
```

Comment:
I'd just overwrite the input values, makes it more readable:
```suggestion
        y_new = np.array(y_new)
```

Reply:
Changed `actual_values` and `y_val` to `y_new` and `X_val` to `X_new`.